PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalAccount.kt

```diff
@@ -18,7 +19,8 @@ import org.json.JSONObject
  * @property paymentType Payment type from original PayPal request. Either "billing-agreement" or
  * "single-payment"
  */
-internal data class PayPalAccount(
```

Comment:
Curious - Was there an access issue in the tests when converting to Kotlin?

Reply:
Yeah there's an integration test in Core that uses PayPalAccount for some reason. We may be able to audit right before we go GA to make sure.